installer/create-cluster-eks.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+# Copyright (C) 2016-2019 Lightbend Inc. <https://www.lightbend.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage:
+# create-cluster-eks.sh [CLUSTER-NAME]
+if [ $# -eq 0 ]
+  then
+    echo "No cluster name supplied"
+    echo "Usage: create-cluster-eks.sh [CLUSTER-NAME]"
+    exit 1
+fi
+
+CLUSTER_NAME=$1
+
+# Create cluster
+# https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html
+eksctl create cluster \
+  --name $CLUSTER_NAME \
+  --version 1.14 \
+  --region us-east-1 \
```

Comment:
Note for the future. Eventually the K8s version and cluster region should probably become parameters. Probably too big a change for this Pr but good to keep in mind.

Reply:
I can have a follow-up PR once we merge down this guy